src/main/java/org/bihealth/mi/easybus/implementations/email/BusEmail.java

```diff
@@ -69,25 +70,23 @@
     private Thread          thread;
     /** Stop flag */
     private boolean         stop = false;
-  
+    /** Sleep time a thread waits to send an e-mail */
+    private int threadSleepTime;
+ 
     /**
      * Creates a new instance
      * @param connection
      * @param millis - interval in milliseconds in which messages are polled
      */
     public BusEmail(ConnectionEmail connection, int millis) {
         this.connection = connection;
+        this.threadSleepTime = millis;
         this.thread = new Thread(new Runnable() {
             @Override
             public void run() {
                 try {
                     while (!stop) {
-                        try {
-                            receiveEmails();
-                        } catch (BusException e) {
-                            // Stop thread
-                            throw new RuntimeException(e);
-                        }
```

Comment:
I'm not convinced that it is a good idea to no throw/catch BusException anymore. Please explain.

Reply:
If an exception is thrown in the current coding (e.g. internet ceases for a small period of time), the receiving e-mail loop will be ended and no receiving will be done anymore. This was a problem in the peformance evaluation runs.
In this proposal the exception might be logged or something but the loop will not stop and in the next interval receiveMail will be started again.
Please advise how to solve the problem otherwise.

Reply:
Or would it be a soultion to throw the exception after a couple of consecutive receiveMail executions have failed as proposed for the send mail function?